From 4e1df02354492d874aff04135a4ad957421b0ed1 Mon Sep 17 00:00:00 2001
From: Alex Gonzalez <alex.gonzalez@digi.com>
Date: Sat, 14 Feb 2015 21:59:03 +0100
Subject: [PATCH] wandboard: Change board info.

Signed-off-by: Alex Gonzalez <alex.gonzalez@digi.com>
---
 board/wandboard/wandboard.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/wandboard/wandboard.c b/board/wandboard/wandboard.c
index 3c8b7a5d2d0a..a466d4c74b8f 100644
--- a/board/wandboard/wandboard.c
+++ b/board/wandboard/wandboard.c
@@ -404,7 +404,7 @@ int board_init(void)
 
 int checkboard(void)
 {
-	puts("Board: Wandboard\n");
+	puts("Board: Wandboard custom\n");
 
 	return 0;
 }
